cmd/thriftrw-list-deps/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"go.uber.org/thriftrw/compile"
+)
+
+var opts struct {
+	RelativeTo string `long:"relative-to" description:"If specified, output paths will be relative to this directory"`
+	Args       struct {
+		ThriftFile string `positional-arg-name:"file" description:"Path to the Thrift file"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+// listDependentThrifts lists the Thrift files that the given Thrift file depends on or includes, both directly (through imports) and
+// indirectly (through transitive imports).
+//
+// The returned file paths are absolute, unless relativeTo parameter is given, in which case the paths are relative to
+// the relativeTo directory.
+func listDependentThrifts(input string, relativeTo string) ([]string, error) {
+	var deps []string
+
+	module, err := compile.Compile(input)
+	if err != nil {
+		return nil, fmt.Errorf("could not compile %q: %v", input, err)
+	}
+
+	module.Walk(func(mod *compile.Module) error {
```

Comment:
This method returns the error from the provided function. That should be handled.